charts/kubewarden-controller/templates/webhooks.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  annotations:
+    {{- if .Values.tls.useCertManager }}
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/kubewarden-controller-serving-cert
```

Comment:
Shouldn't that be `{{ include "kubewarden-controller.fullname" . }}-serving-cert` ?

Reply:
Indeed, thanks! Done in https://github.com/kubewarden/helm-charts/pull/34/commits/f8f6ebf05da963f1d5670a5e853fcc6e2c1c21ef.